src/compressed_tensors/compressors/model_compressors/model_compressor.py

```diff
@@ -535,6 +535,12 @@ def map_modules_to_quant_args(model: Module) -> Dict[str, QuantizationArgs]:
             if submodule.quantization_scheme.weights is not None:
                 name = fix_fsdp_module_name(name)
                 quantized_modules_to_args[name] = submodule.quantization_scheme.weights
+                if submodule.quantization_scheme.input_activations is not None:
+                    weight_args = quantized_modules_to_args.get(name)
+                    quantized_modules_to_args[name] = (
+                        weight_args,
+                        submodule.quantization_scheme.input_activations,
+                    )
```

Comment:
Could we update the typing of this function to reflect the change + describe in the docstring when will we have a mapping from str -> tuple vs str -> QuantizationArgs